src/MainWindow.ui

```diff
@@ -1288,6 +1290,24 @@
     <string>Export as &amp;AMF...</string>
    </property>
   </action>
+  <action name="fileActionExportX3D">
+   <property name="icon">
+    <iconset resource="../openscad.qrc">
+     <normaloff>:/images/X3D.png</normaloff>:/images/X3D.png</iconset>
```

Comment:
They are hand-drawn. There are some SVGs buried in a PR somewhere (#1214). I don't think icons are necessary unless these icons make it into a toolbar.